Automattic/newspack-theme
# Build out archive styles

### All Submissions:






### Changes proposed in this Pull Request:

This PR is a first pass at building out the default archive styles, mocked up here: 

https://cloudup.com/iTtVYsGj34T

**What it does:**

* Displays the taxonomy description (if there is one)
* Styles the page heading text to match the mockup
* Makes sure the first archive item is slightly larger than the rest, and displays an excerpt (the others do not).
* Aligns the featured image to the left for the other articles, if there is one.

**What it doesn't do**

* Address the content interspersed between the archive items (ads and CTAs)
* Get the font sizing quite right -- how the font sizes are calculated needs to be updated to better work with the designs.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Pick a category with more than one page of posts; give it a description.
3. View the archive page for the category. Right now things don't need to be pixel perfect (as other changes to the theme -- like to the font sizing -- will affect this page and change fine details) but the main things to check for are the page title styles, the first story styles, and making sure things react well on smaller screens (with featured images on all posts moving above their titles).

### Other information:
